src/app/features/settings-dropdown/settings-dropdown.tsx

```diff
@@ -166,6 +167,15 @@ export function SettingsDropdown() {
               Lock
             </MenuItem>
           )}
+          {!IS_PROD_ENV && (
```

Comment:
Maybe we can avoid this and just get people that need it to bookmark the URL? 

Just as I don't think we should mix sandbox stuff with real production code

Reply:
it's just for convenience. I believe there is no problem to add menu item there